CITATION

```diff
@@ -0,0 +1,126 @@
+When using CellProfiler, please cite one of the following papers.
+See also: http://cellprofiler.org/citations/
+
+
+CellProfiler for cells:
+
+@Article{Carpenter2006,
+author="Carpenter, Anne E.
```

Comment:
Please indent all fields between `{...}`s by one tab/two spaces.

Reply:
Do you know a linter? I'm using Atom right now, and the auto-format doesn't work on those snippets. Copying verbatim did seem a good idea at the time, but which formatting standard should it be made to adhere to now?

Reply:
I don't know of a linter specific to this formatting. I'll fix the formatting before the merge.

Reply:
OK, thank you! If I find a BibTeX linter, I'll let you know.